Makefile

```diff
@@ -33,6 +33,8 @@ GOFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/minishift/ | grep $(REPOPATH)
 PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
 
+MINISHIFT_GITHUB_API_TOKEN = $(shell echo $MINISHIFT_GITHUB_API_TOKEN)
```

Comment:
@praveenkumar sorry misunderstood you here. I am also interested to know how this variable is being used in Makefile.